ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource

```diff
@@ -147,18 +147,28 @@ Open Pipeline Detail
     Capture Page Screenshot
 
 Open Pipeline Run
-    [Documentation]    Open the Pipeline Run detail
-    [Arguments]    ${pipeline_run_name}
-    Wait Until Page Contains Element    xpath=//button[text()='Data Science Pipelines']
-    Click Element    xpath=//button[text()='Data Science Pipelines']
-    Wait Until Page Contains Element    xpath=//a[text()='Runs']
+    [Documentation]    Open the Pipeline Run detail. The menu hold a state and if you click twice in the same element,
+    ...    it won't work. Set force_reload equals True
+    [Arguments]    ${pipeline_run_name}    ${force_reload}=${FALSE}
+    ${present}=  Run Keyword And Return Status    Element Should Be Visible   xpath=//a[text()='Runs']
+    # We can run Open Pipeline Run step multiple times. If the menu is expanded, then it will colapse
+    IF    ${present} == ${FALSE}
+        Wait Until Page Contains Element    xpath=//button[text()='Data Science Pipelines']
+        Click Element    xpath=//button[text()='Data Science Pipelines']
+        Wait Until Page Contains Element    xpath=//a[text()='Runs']
+    END
     Click Element    xpath=//a[text()='Runs']
-    Wait Until Page Contains Element    xpath=//span[text()='Triggered']
+    IF    ${force_reload} == ${TRUE}
+        Reload Page
+    END
```

Comment:
you click on Runs and nothing happens right? hence you need the page reload, is that correct?